build_tools/github_actions/test_executable_scripts/test_rocwmma.py

```diff
@@ -33,10 +33,10 @@
     # The emulator regression tests are very fast.
     # If we need something even faster we can use "/smoke" here.
     test_subdir = "/regression"
-    timeout = "300"
+    timeout = "900"
```

Comment:
This PR has conflicts to resolve now, fyi.

https://github.com/ROCm/TheRock/commit/d4fd7cbb08152a4d8960b50d78be742afa3ff0a2 increased the timeout to 720. Do we still want 900?

Reply:
I suspect the 12min timeout will be sufficient.  Once the refs are bumped in rocm-libraries we should have a better idea.  I'll keep this open until we know it's not necessary.